unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PluginSystem/PluginSystemFactory.cs

```diff
@@ -0,0 +1,22 @@
+using DCL.Tutorial;
+
+namespace DCL
+{
+    public static class PluginSystemFactory
+    {
+        public static PluginSystem Create()
+        {
+            var pluginSystem = new PluginSystem();
+
+            pluginSystem.Register(() => new DebugPluginFeature());
+            pluginSystem.Register(() => new ShortcutsFeature());
+            pluginSystem.RegisterWithFlag(() => new BuilderInWorldPlugin(), "builder_in_world");
+            pluginSystem.RegisterWithFlag(() => new TutorialController(), "tutorial");
```

Comment:
We should use only one naming style for plugin classes for better readability, in this case, either `TutorialPlugin` or `TutorialFeature`, and do the same for the rest

Reply:
Agreed 100%. I think that some classes were named after `Feature` because we had `PluginFeature` but now we have just `IPlugin` /shrug.